client/src/partials/journal/modals/journalPosting.modal.html

```diff
@@ -0,0 +1,65 @@
+<form name="JournalPosterForm" ng-submit="JournalPosterModalCtrl.submit()" bh-form-defaults novalidate>
+
+  <div class="modal-header">
+    <ol class="headercrumb">
+      <li class="static">{{ ::"TREE.POSTING_JOURNAL" | translate }}</li>
+      <li class="title">
+        {{ ::"POSTING_JOURNAL.TRIAL_BALANCE.TITLE" | translate }}
+      </li>
+    </ol>
+  </div>
+
+  <div class="modal-body">
+      <p>
+        <div class="row">
+          <div class="col-xs-4 pull-left">
+            <button type="button" ng-show="JournalPosterModalCtrl.feedBack.hasError || JournalPosterModalCtrl.feedBack.hasWarning" class="btn {{JournalPosterModalCtrl.cssClass}}" ng-click="JournalPosterModalCtrl.openErrorViewerModal()" data-method="show">
+              <i class="fa fa-bars"></i>
+              {{"FORM.BUTTONS.VIEW_ERRORS" | translate }}
+            </button>
+          </div>
+
+          <div class="pull-right">
+            <button id="switchViewID" type="button" class="btn btn-default" ng-click="JournalPosterModalCtrl.switchGroup()" data-method="switch">
+              <i class="fa fa-retweet"></i>
+              {{JournalPosterModalCtrl.groupingDetail.key | translate }}
+            </button>
+          </div>
+        </div>
+      </p>
+
+    <div
+      id="transaction-grid"
+      ui-grid="JournalPosterModalCtrl.gridOptions"
+      ui-grid-auto-resize
+      ui-grid-grouping
+      ng-if="JournalPosterModalCtrl.errors"
```

Comment:
I would put the `ng-if` on a div wrapping this div.  That way, if it is false, we can be _sure_ the DOM does not compile.